doc/plugins.md

```diff
@@ -2,33 +2,33 @@
 
```

Comment:
IMHO it’s a bit redundant to keep both this list and awesome-remark. I think it would be better to remove this file and update awesome-remark.

Reply:
With `awesome-remark`, the story is that it’s a list of the *most awesome* projects. It’s curated. The entries there are *recommend*.
I think the goal is, not to flood users with everything that’s possible, but to inspire with some possibilities.

This list is not that. It’s a list of all the plugins that folks ever wanted to highlight. A lot of duplicates. Not necessarily recommended. 
I mean, I sometimes go through them to remove stuff that hasn’t been updated or used in say 3+ years, but generally its everything that’s a plugin.
I think the goal here is a way to browse essentially all plugins (useful when a new breaking release comes out, e.g., for remark 13 I went through all plugins: https://github.com/remarkjs/remark/pull/536#ref-issue-714079690), and a place to show your new plugin (and a moment for us to look through the code and recommend some changes).

With search comes a lot of crap. Gatsby copies. Forks. Things that aren’t plugins at all.

Reply:
so,
- redundant? Yes, there is some duplication, not too much though
- only having an awesome-remark? “awesome” is supposed to be the most awesome, cool, popular, so it wouldn’t match 1-to-1